zephyr/dts/riscv/tinyvision/tinyclunx33_rtl_1_2.dtsi

```diff
@@ -1,9 +1,10 @@
 /*
- * Copyright (c) 2025 tinyVision.ai Inc.
+ * Copyright (c) 2024 tinyVision.ai Inc.
```

Comment:
Maybe we do like NXP and don't have a year in the copyright to avoid getting tired in constantly updating it :]

Reply:
:)

Reply:
it's in the blame anyway.